sycl/test-e2e/KernelFusion/queue-shortcut-functions.cpp

```diff
@@ -80,7 +80,7 @@ int main() {
   // should be cancelled as a dependency with an event to be fused is found.
 
   // CHECK:      FusionStartPoint = 0:
-  // CHECK-NEXT: WARNING: Not fusing 'copy usm' command group. Can only fuse device kernel command groups.
+  // CHECK: WARNING: Not fusing 'copy usm' command group. Can only fuse device kernel command groups.
```

Comment:
Can you please provide a rationale why this change is necessary? 

Is there any additional output between the two lines?

Reply:
When L0 Adapter is being used, the new L0 loader always reports the loader init in a single print. That broke this test so this needs to be changed to check for this line instead of exactly after this line.

Reply:
Ok, thanks for the explanation.